### PR TITLE
Fixed notification nums on navbar in admin dashboard

### PR DESCRIPTION
**Pull request by jacobduncan00**
- Simple fix
- Someone used query.filter() instead of query.filer_by()
- Need more thorough testing for all cases inside an API route
- We neglected to test this 'unseen' condition and assumed it worked when all we tested was the 'all' condition

### CODE REVIEW

File: api/admin.py [hunk 1]
```diff
@@ -154,8 +154,7 @@ def get_messages(message_filter: str = 'all'):
 
     # For querying just messages labelled as unseen.
     elif message_filter == 'unseen':
-        messages = ContactFormMessage.query.filter(was_seen=False)
-        # TODO: label as seen?
+        messages = ContactFormMessage.query.filter_by(was_seen='f').all()
```
Comment (jventura1738) [inline api/admin.py:157]: Any particular reason for 'f' instead of False?

**Review by jventura1738 — APPROVED** [review]
Looks good, need rest of team to test now

File: api/admin.py (hunk 1 above)
Comment (jacobduncan00) [reply api/admin.py:157]: No, I just saw 'f' and 't' in the table and thought I had to use that

Comment (jventura1738) [reply api/admin.py:157]: No worries, works either way.  Was just curious

**Pull request merged**